requirements.txt

```diff
@@ -1,2 +1,4 @@
 Django==5.1.*
-python-decouple==3.8
\ No newline at end of file
+python-decouple==3.8
+pylint==3.3.*
+pylint-django==2.6.*
```

Comment:
How about adding an empty line at the end? (In other files as well)
It prevents irrelevant diffs like here.